packages/@aws-cdk/aws-eks-v2-alpha/test/integ.eks-cluster-ipv6.js.snapshot/asset.a2c21e49681db6d38ad6c55b325a1168522925850c5e1e96b8f94f9bada84cef/fargate.ts

```diff
@@ -0,0 +1,120 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as EKS from '@aws-sdk/client-eks';
+import { ResourceHandler } from './common';
+
+const MAX_NAME_LEN = 63;
+
+export class FargateProfileResourceHandler extends ResourceHandler {
```

Comment:
Are the integration tests updated yet in this PR? As I assume these are custom resource handlers

Reply:
Good catch. The actual custom resource handler should be removed because it's not needed anymore.

Reply:
Updated the integration tests